src/lib/test-handler.js

```diff
@@ -65,6 +68,22 @@ export default class TestHandler {
       });
   }
 
+  mergeLogs() {
+    let testResults = [];
+    let mergedFileName = path.join(this.options.logDir, 'merged', 'results.json');
+    let logFilePaths = fs.readdirSync(this.options.logDir);
+    logFilePaths.forEach((logFilePath) => {
+      if (_.endsWith(logFilePath, '.json')) {
+        testResults = _.concat(
+          testResults,
+          JSON.parse(fs.readFileSync(path.join(this.options.logDir, logFilePath), 'utf8'))
```

Comment:
We're using fs-extra, which adds some convenient things, one of which being readJson/readJsonSync https://github.com/jprichardson/node-fs-extra#readjsonfile-options-callback

Alternatively, you can just `require(x.json)` and it will so the same.